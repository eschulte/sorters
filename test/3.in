7 6 4 9 7 9 2 9 6 3