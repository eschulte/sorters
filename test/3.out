2 3 4 6 6 7 7 9 9 9
